src/TableColumn/ActionButtons.php

```diff
@@ -100,18 +100,20 @@ public function addButton($button, $action = null, $confirm = false, $isDisabled
      * load contents through $callback. Will pass a virtual page.
      *
      * @param \atk4\ui\View|string $button
-     * @param string               $title
+     * @param string|array         $defaults modal title or modal defaults array
      * @param callable             $callback
      * @param \atk4\ui\View        $owner
      * @param array                $args
      *
      * @return \atk4\ui\View
      */
-    public function addModal($button, $title, $callback, $owner = null, $args = [])
+    public function addModal($button, $defaults, $callback, $owner = null, $args = [])
     {
         $owner = $owner ?: $this->owner->owner;
 
-        $modal = \atk4\ui\Modal::addTo($owner, [compact('title'), 'appStickyCb' => true]);
+        $defaults = [is_array($defaults) ? $defaults : ['title' => $defaults], 'appStickyCb' => true];
```

Comment:
Array merge is probably needed, to this should be more like:
```
if (is_string($defaults)) {
    $defaults = ['title' => $defaults];
}

$defaults['appStickyCb'] = true;
```

Reply:
It works like it is. I did not dig deeper enough to see why it works :)
I also prefer array_merge